src/ui/components/instantiate/Step2.tsx

```diff
@@ -91,6 +91,7 @@ export function Step2() {
               registry={metadata.registry}
               setArgValues={setArgValues}
               argValues={argValues}
+              className="argument-form"
```

Comment:
some components don't pass props down to the actual DOM element so `data-cy` is lost. i tried to add the data element where possible, and use css classes in some cases to avoid prop drilling.